### PR TITLE
[FEATURE] Mobile Interactive Menu

### PR DESCRIPTION
**Pull request by Andyveloper**
# Additions
- JS script to make the menu interactive
- CSS classes to be able to call them in the JS code

# Changes 
- Changed some IDs to classes to use the classList method on JS.
- Fixed Stylinters

## Please take into account
- Please read the code before and test it out before deciding its working or not, thank you.

Thank you

### CODE REVIEW

**Review by phelian23 — COMMENTED** [review]
Hi @Zeraltz, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the linter is failing. Kindly fix the linter error and resubmit when all checks are passed

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by Andyveloper** [comment]
> Hi @Zeraltz,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that the linter is failing. Kindly fix the linter error and resubmit when all checks are passed
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **Does not count** into the code reviews limit._

Did you read the 'PLEASE TAKE INTO ACCOUNT' section of the PR?, if you didn't please take a look at it

File: index.html [hunk 1]
```diff
@@ -16,14 +21,17 @@
...
     </div>
-    <nav>
-      <img id="burgermenu" src="./img/burger.png" alt="menu icon">
-      <div id="menu">
+    <nav class="indexPriority">
+      <button class="burgermenu">
+        <i class="menuIcon material-icons">menu</i>
+        <i class="closeIcon material-icons">close</i>
+      </button>
+      <div class="menu">
```
Comment (OLIPLICHE) [inline index.html:29]: - [ ] Please make sure when the user clicks any of the mobile menu options(Portfolio, About...), the correct part of the page is displayed

- mn

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Andyveloper** [comment]
Already made all the changes and requested for a second review, thanks a lot

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Zeraltz 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**